lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -1,4 +1,6 @@
 import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
AFAIK everywhere else we keep the `pyplot` import local to the plotting functions, might want to move this to keep consistent

Reply:
👍

Reply:
> looks good. there's just the one small comment about importing pyplot within the function rather than the full module as we do in other places

was a rebase mishap (the import ended up in both places)